2.ServiceTracker_App/ComponentSchematic/tracker-db-component.yaml

```diff
@@ -1,54 +1,45 @@
 apiVersion: core.oam.dev/v1alpha1
```

Comment:
I agree. I do like the idea of having this version merged (since it is going to master) and update this (and other examples) to use v1alpha2. Today, AFAIK, we don't have a v1alpha2 implementation we can run this example.